codecov.yml

```diff
@@ -0,0 +1,11 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+comment:
+  require_changes: yes
```

Comment:
This will make the comment bot less noisy I've noticed, hard to say for sure until it's operating on master branch which has a base report to compare to. May need to be changed in the future.

Docs here: https://docs.codecov.io/docs/pull-request-comments